Doc/library/logging.rst

```diff
@@ -1019,6 +1019,14 @@ information into logging calls. For a usage example, see the section on
       'extra'. The return value is a (*msg*, *kwargs*) tuple which has the
       (possibly modified) versions of the arguments passed in.
 
+   .. attribute:: manager
+
+      Delegates to the underlying :attr:`!manager`` on *logger*.
```

Comment:
Why the reference is silenced? Is the `manager` attribute documented? If not, then the proper fix is to document it.

Reply:
I created gh-108198 so we don't forget about it.